src/tests/simple_pipeline_arrays.c

```diff
@@ -373,7 +373,7 @@ run_pipeline(daos_pipeline_t *pipeline)
 
 		rc = daos_pipeline_run(coh, oh, pipeline, DAOS_TX_NONE, 0,
 				       NULL, &nr_iods, &iod, &anchor, &nr_kds,
-				       kds, sgl_keys, sgl_recs ,NULL, &scanned,
+				       kds, sgl_keys, sgl_recs ,NULL, &stats,
```

Comment:
(style)  space prohibited before that ',' (ctx:WxV)
(style)  space required after that ',' (ctx:WxV)